Dockerfile-jmx

```diff
@@ -14,11 +14,14 @@ ENV DOCKER_DD_AGENT=yes \
     DD_CONF_PROCFS_PATH="/host/proc"
 
 # Install the Agent
-RUN echo "deb http://apt.datadoghq.com/ stable main" > /etc/apt/sources.list.d/datadog.list \
+RUN apt-get update \
+ && apt-get install --no-install-recommends -y gnupg dirmngr \
+ && echo "deb http://apt.datadoghq.com/ stable main" > /etc/apt/sources.list.d/datadog.list \
  && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A2923DFF56EDA6E76E55E492D3A80E30382E94DE \
  && apt-get update \
- && apt-get install --no-install-recommends -y openjdk-7-jre-headless \
  && apt-get install --no-install-recommends -y datadog-agent="${AGENT_VERSION}" \
+&& apt-get install --no-install-recommends -y openjdk-8-jre-headless \
```

Comment:
Why not install everything on a one-liner like: 
```bash
apt-get install --no-install-recommends -y openjdk-8-jre-headless ca-certificates ...
```

Reply:
openjdk-8-jre-headless is on its own line so the diff between `Dockerfile` and `Dockerfile-jmx` is constant, ca-certs could go back to the agent line indeed